### DISCUSSION TITLE
How to get rid of daily reminder?

### DISCUSSION THREAD

**Discussion by sm8ps** [super-productivity/super-productivity#2765] [Q&A]
I remember filling out some kind of evaluation of my working day upon finishing the app. At the time, I had some shopping to do which I put down under "What should I do to make my working more efficient?" or something the like. That was about three weeks ago and now the reminder pops up every day upon starting the app. 

![Screenshot_20230902_070916-1](https://github.com/johannesjo/super-productivity/assets/3989424/ea09c95d-cdab-47e9-8561-db0ee0ec8c94)

Either way if I click "Done" or "Dismiss" (not in the image but appears later), the reminder reappears the next day. Please can anybody help me getting rid of this groundhog day shopping nightmare? Nothing is registered under scheduled or repeating tasks.

**Comment by johannesjo** [comment]
If you click done it should be gone for good unless you put the same thing in the evaluation sheet in "What could you do to improve tomorrow" at the end of the day. If it isn't, it's a bug.

**Comment by sm8ps** [comment]
Thanks a lot, @johannesjo for your definitive answer. Indeed, the reminder has not shown up anymore after I have read your words out aloud to the app and showing it this discussion here. Hearing his master's voice -- so to say -- it shrieked back and promised to behave from now on.
Seriously, though, I believe the issue may have been connected to #2110 that I observed at around the same time.
